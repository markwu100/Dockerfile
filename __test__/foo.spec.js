const foo = require('../index.js');

describe('docker test', () => {
  it('should throw snapshot error', () => {
    expect(foo).toMatchSnapshot();
  });
});
