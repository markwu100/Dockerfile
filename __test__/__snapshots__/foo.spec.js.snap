// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`docker test should throw snapshot error 1`] = `2018-02-13T16:01:14.325Z`;
